Monika After Story/game/script-topics.rpy

```diff
@@ -4910,3 +4910,218 @@ label monika_asks_family:
+#This is where I need the code or a way to somehow jump to another part of this topic#
+                                "Fletcher":
+                                    m "That's Right!"
+                                    m "I guess you did watch it."
+                                    m "Sorry for doubting you [player]."
+                                    menu:
+                                        "It's fine.":
+                                            m "Sorry I ever doubted you [player]."
+                                            m "So, what'd you think of the movie?"
+                                            menu:
+                                                "Pretty good."
+                                                    m "Yeah, It was pretty good."
+                                                    m "I really liked the jazz orchestra at the end!"
+                                                    m "The music was good too."
+                                                    m "It's not like that movie is gonna be my all time favourite."
+                                                    m "But I enjoyed watching it."
+                                                    m "Maybe we could even watch it together some other time!"
+                                                    menu:
+                                                        "Fine by me.":
+                                                            m "You and me, watching a movie together."
+                                                            m "That'd be something."
+                                                            m "I can't wait!"
+                                                        "Maybe something else.":
+                                                            m "Yeah, maybe it's not the type of movie anyone would watch again and again."
+                                                            m "There are other better ones out there."
+                                                            m "Maybe we could find one together [player]?"
+                                                            m "You and me, watching a movie together."
+                                                            m "Wouldn't that be fun?"
+                                                            m "Even just spending time with you is enough to make my heart pound."
+                                                            m "{i}Ehehe{/i}~"
+                                                        "I'd be too busy watching you instead.":
+                                                            m "That's so cheesy."
+                                                            m "You're making me blush."
+                                                            m "Calm down Monika, Calm down..."
+                                                "Not my type."
```

Comment:
missing `:` at the end of `"Not my type."`

Reply:
done